json.js

```diff
@@ -6,12 +6,15 @@ const jsonStringify = require('fast-safe-stringify');
 
 /*
  * function replacer (key, value)
- * Handles proper stringification of Buffer output.
+ * Handles proper stringification of Buffer and bigint output.
  */
 function replacer(key, value) {
-  return value instanceof Buffer
-    ? value.toString('base64')
-    : value;
+  if (value instanceof Buffer)
+    return value.toString('base64');
+  // eslint-disable-next-line valid-typeof
+  if (typeof value === 'bigint')
+    return value.toString();
```

Comment:
Seems bigint may have a toJSON function?  Is there any difference between toString and toJSON()?
https://github.com/tc39/proposal-bigint/issues/146

Reply:
It doesn't have it yet.
```
> const a = 1n
undefined
> a.toString()
'1'
> a.toJSON()
TypeError: a.toJSON is not a function
```

Reply:
Ah, stackoverflow misled me.  Thanks for checking.